src/Compilers/CSharp/Portable/Symbols/Tuples/TupleFieldSymbol.cs

```diff
@@ -115,6 +116,11 @@ internal override DiagnosticInfo GetUseSiteDiagnostic()
             return _underlyingField.GetUseSiteDiagnostic();
         }
 
+        internal override void ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
```

Comment:
>ForceComplete [](start = 31, length = 13)

It feels like we should also override ```HasComplete``` and possible some other completion related APIs